src/main/java/essenmakanan/ingredient/IngredientUnit.java

```diff
@@ -0,0 +1,22 @@
+package essenmakanan.ingredient;
+
+public enum IngredientUnit {
+    GRAM("g"),
+    KILOGRAM("kg"),
+    MILLILITER("ml"),
+    LITER("l"),
+    TEASPOON("tsp"),
+    TABLESPOON("tbsp"),
+    CUP("cup"),
+    PIECE("pc");
```

Comment:
Good use of abbreviation. When is it used though? If it is not used is it necessary? Is it for clarity because user input is in that format?